python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1769,25 +1766,25 @@ def computeAltMaskList(self, tempExpRefList, maskSpanSets):
 
         return altMaskList
 
-    def _filterArtifacts(self, spanSetList, epochCountImage, maxNumEpochs=None):
+    def _filterArtifacts(self, spanSetList, epochCountImage):
         maskSpanSetList = []
         x0, y0 = epochCountImage.getXY0()
         for i, span in enumerate(spanSetList):
             y, x = span.indices()
             counts = epochCountImage.array[[y1 - y0 for y1 in y], [x1 - x0 for x1 in x]]
-            idx = numpy.where((counts > 0) & (counts <= maxNumEpochs))
+            idx = numpy.where((counts > 0) & (counts <= self.config.maxNumEpochs))
```

Comment:
You should avoid using `numpy.where` whenever you can (it's expensive), and just use a boolean array. Then, `len(idx[0])` below simply becomes `array.sum()`.